Prox/Prox/PlaceDetails/PlaceDetailsCardViewController.swift

```diff
@@ -113,13 +113,13 @@ class PlaceDetailsCardViewController: UIViewController {
         if let lastTravelTimes = place.lastTravelTime {
             self.cardView.updateTravelTimesUI(travelTimes: lastTravelTimes)
         } else {
-            self.cardView.travelTimeView.loadingSpinner.startAnimating()
+            self.cardView.travelTimeView.isLoading = true
         }
         if let location = location {
             place.travelTimes(fromLocation: location, withCallback: { travelTimes in
-                self.cardView.travelTimeView.loadingSpinner.stopAnimating()
-                guard let travelTimes = travelTimes else { return }
-                self.cardView.updateTravelTimesUI(travelTimes: travelTimes)
+//                self.cardView.travelTimeView.isLoading = false
```

Comment:
was it your intention to leave these lines commented out? With these lines commented I cannot see how the spinner will every stop spinning?